docs/reference/deploy/master-config-reference.rst

```diff
@@ -241,30 +241,6 @@ otherwise active (as defined by the ``notebook_idle_type`` option in the :ref:`t
 
 The resource manager used to acquire resources. Defaults to ``agent``.
 
```

Comment:
I don't think this part got reverted?
https://github.com/determined-ai/determined/pull/9122

Reply:
you are correct; adding that

Reply:
I don't see the helm/charts/determined/values.yaml part reverted? I consider that part of the public facing docs for this feature

Reply:
that will have to be a code PR